src/qt/radiobut.cpp

```diff
@@ -42,7 +44,18 @@ bool wxRadioButton::Create( wxWindow *parent,
     m_qtRadioButton = new QRadioButton( parent->GetHandle() );
     m_qtRadioButton->setText( wxQtConvertString( label ));
 
-    return QtCreateControl( parent, id, pos, size, style, validator, name );
+    bool createdOk = QtCreateControl( parent, id, pos, size, style, validator, name );
```

Comment:
If creation failed should it still add the button to a group? Shouldn't it just return `false` here?
...and in consequence return `true` at the end?

Reply:
Yes, you're right. The code has been amended to do that.